libsolidity/ast/ExperimentalFeatures.h

```diff
@@ -45,6 +46,7 @@ static const std::map<std::string, ExperimentalFeature> ExperimentalFeatureNames
 {
 	{ "SMTChecker", ExperimentalFeature::SMTChecker },
 	{ "ABIEncoderV2", ExperimentalFeature::ABIEncoderV2 },
+	{ "v0.5.0", ExperimentalFeature::V050 },
```

Comment:
Not sure which one we're going for, `v0.5.0`, `v050`, `V0.5.0`, .. ?

Reply:
I think something like `v0.5.0` is fine, but do we want to have the trailing `0`? Perhaps we should also define these features somewhere, i.e. what exactly does it activate?

Reply:
I think we can have a documentation section about them if needed, but wouldn't put `v0.5.0` in there, it is a bit more like having an `#ifdef` so we can merge breaking changes in preparation for the breaking release.